docs/eventing/samples/writing-a-source/04-publish-to-cluster.md

```diff
@@ -196,4 +196,4 @@ kubectl get pods -n sample-source-system
 Now you can use the verification procedure outlined above in
 [Create a sample source](#create-a-sample-source).
 
-Next: Dispatching Events
+(TODO) Next: Dispatching Events
```

Comment:
```suggestion
Next: Dispatching Events
```

Reply:
No visible TODOs in docs. Could just delete this line.

Reply:
will need to wipe all the other references to this in the corresponding files. will leave as is for clarity since this is non existent and is better off with a visible todo in the file